examples/trials/ga_squad/README.md

```diff
@@ -75,12 +77,70 @@ In the "trial" part, if you want to use GPU to perform the architecture search,
 
 `trialConcurrency` is the number of trials running concurrently, which is the number of GPUs you want to use, if you are setting `gpuNum` to 1.
 
-## submit this job
+### submit this job
 
 ```
 nnictl create --config ~/nni/examples/trials/ga_squad/config.yml
 ```
 
+## Run this example on OpenPAI
+
+Due to the memory limitation of upload, we only upload the source code and complete the data download and training on OpenPAI. This experiment requires sufficient memory that `memoryMB >= 32G`, and the training may last for several hours.
+
+### Update configuration
+Modify `nni/examples/trials/ga_squad/config_pai.yaml`, here is the default configuration:
+
+```
+authorName: default
+experimentName: example_ga_squad
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+#choice: local, remote, pai
+trainingServicePlatform: pai
+#choice: true, false
+useAnnotation: false
+#Your nni_manager ip
+nniManagerIp: 10.10.10.10
+tuner:
+  codeDir: ../../tuners/ga_customer_tuner
+  classFileName: customer_tuner.py
+  className: CustomerTuner
+  classArgs:
+    optimize_mode: maximize
+trial:
+  command: chmod +x ./download.sh && ./download.sh && python3 trial.py
+  codeDir: .
+  gpuNum: 0
+  cpuNum: 1
+  memoryMB: 32869
+  #The docker image to run nni job on pai
+  image: msranni/nni:latest
+  #The hdfs directory to store data on pai, format 'hdfs://host:port/directory'
+  dataDir: hdfs://10.10.10.10:9000/username/nni
+  #The hdfs directory to store output data generated by nni, format 'hdfs://host:port/directory'
+  outputDir: hdfs://10.10.10.10:9000/username/nni
+paiConfig:
+  #The username to login pai
+  userName: username
+  #The password to login pai
+  passWord: password
+  #The host of restful server of pai
+  host: 10.10.10.10
+```
+
+Please change the default value to your personal account and machine information. Including `nniManagerIp`, `dataDir`, `outputDir`, `userName`, `passWord` and `host`.
+
+In the "trial" part, if you want to use GPU to perform the architecture search, change `gpuNum` from `0` to `1`. You need to increase the `maxTrialNum` and `maxExecDuration`, according to how long you want to wait for the search result.
+
+`trialConcurrency` is the number of trials running concurrently, which is the number of GPUs you want to use, if you are setting `gpuNum` to 1.
+
+### submit this job
+
+```
+nnictl create --config ~/nni/examples/trials/ga_squad/config_pai.yml
```

Comment:
`~/nni/examples/trials/ga_squad/config_pai.yml` assume that user clone nni under $HOME directory

Reply:
Sorry, not clear about the question? It's just an example of the path for config_pai.yml, also consistent with other examples.